asyncapi/salesforce/service_types.bal

```diff
@@ -14,28 +14,29 @@
 // specific language governing permissions and limitations
 // under the License.
 
-# Salesforce event service object.
-public type StreamingEventService service object {
+# Triggers when a new event related to Salesforce records is received.
```

Comment:
Doesn't it need to be corrected as `...event related to Salesforce record is received`?